src/chttpd/src/chttpd_db.erl

```diff
@@ -902,9 +902,10 @@ send_all_docs_keys(Db, #mrargs{} = Args, VAcc0) ->
         _ -> Args#mrargs.doc_options
     end,
     IncludeDocs = Args#mrargs.include_docs,
-    lists:foldl(fun(DocId, Acc) ->
-        OpenOpts = [deleted | DocOpts],
-        Row0 = case fabric2_db:open_doc(Db, DocId, OpenOpts) of
+    OpenOpts = [deleted | DocOpts],
+
+    CB = fun({doc, DocId, Doc}, Acc) ->
```

Comment:
This `{doc, DocId, Doc}` seems a bit awkward. Maybe better as `fun(DocId, Doc, Acc) -> ... end`?

Reply:
Fixed